nifi-nar-bundles/nifi-extension-utils/nifi-processor-utils/src/main/java/org/apache/nifi/processor/util/list/AbstractListProcessor.java

```diff
@@ -375,7 +376,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
                         // If our determined timestamp is the same as that of our last listing, skip this execution as there are no updates
                         if (minTimestampToListMillis.equals(this.lastListedLatestEntryTimestampMillis)) {
                             context.yield();
```

Comment:
nit: I'd prefer if we move this before the return statement

Reply:
Do you mean move the whole if and do not use yielded variable?

Reply:
no I just meant moving this line `context.yield();` in the `if` you introduced, before the return statement.

Reply:
I'd like to move the original 'if' with yield out of the loop closer catch, then I don't need this new 'if' and variable. What do you think?

Reply:
ok, never mind. I'll give variable new name and move context.yield()